src/main/java/Task.java

```diff
@@ -0,0 +1,27 @@
+public class Task {
+    protected String description;
+    protected boolean isDone;
+
+    public Task(String description) {
+        this.description = description;
+        this.isDone = false;
+    }
+
+    public String getStatusIcon() {
+        return (isDone ? "[X]" : "[ ]"); // mark done task with X
+    }
+
+    // Method to mark the tasks as done
+    public void markAsDone(){
```

Comment:
Consider adding a space after the method declaration e.g.
```suggestion
    public void markAsDone() {
```